config/locales/en.yml

```diff
@@ -1867,6 +1867,7 @@ Please follow the instructions there to make your enterprise visible on the Open
     resolve_errors: Please resolve the following errors
     more_items: "+ %{count} More"
     admin:
+      enterprise_limit_reached: "You have reached your enterprise limit. Write to %{contact_email} if you need to increase it."
```

Comment:
Users haven't seen this concept of enterprise limit before. I'd reword this to "You have reached the limit of enterprises per account. ..." or something alike.